src/Files/Strings/af/Resources.resw

```diff
@@ -2538,4 +2643,10 @@ Ons gebruik App Center om tred te hou met programgebruik, foute te vind en ongel
   <data name="SettingsSetAsDefaultFileManagerDescription" xml:space="preserve">
     <value>Hierdie instelling verander stelsellêers en kan onverwagte newe-effekte op jou toestel hê. Die ontwikkelaars neem geen verantwoordelikheid in die geval dat 'n probleem as gevolg daarvan plaasvind nie. Om voort te gaan met hierdie opsie is 'n erkenning van die risiko's wat by hierdie aksie betrokke is. Let asseblief daarop dat die deïnstalleer van Files nie hierdie veranderinge sal ongedaan maak nie en kan verhoed dat jy File Explorer oopmaak, tensy jy hierdie instelling afskakel voordat jy Files van jou toestel verwyder.</value>
   </data>
+  <data name="Extract" xml:space="preserve">
+    <value>Uitreksel</value>
+  </data>
+  <data name="ExtractKeyboardAcceleratorTextOverride" xml:space="preserve">
+    <value>Ctrl+E</value>
+  </data>
```

Comment:
Can you revert the changes to the resw files?

Reply:
I believe the resw files should be reverted now.